proof/refine/AARCH64/Retype_R.thy

```diff
@@ -2397,6 +2397,10 @@ lemma other_objs_default_relation:
                  split: Structures_A.apiobject_type.split_asm)
   done
 
+lemma word_to_tcb_flags_0[simp]:
+  "word_to_tcb_flags 0 = {}"
+  by (clarsimp simp: word_to_tcb_flags_def)
```

Comment:
consider moving higher up, but up to you

Reply:
I actually originally had this marked to be moved but I couldn't find anywhere that made sense for it. It's defined in `Structures_A` and then there aren't any other lemmas about it.